css-writing-modes-3/text-combine-upright/text-decoration-001.htm

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSS Writing Modes Test: text-combine-upright with text-decoration</title>
+    <link rel="author" title="Shinsuke Matsuki" href="mailto:shinsuke.matsuki@access-company.com">
+    <link rel="help" href="http://www.w3.org/TR/css-writing-modes-3/#text-combine-upright">
+    <link rel="match" href="reference/text-decoration-001-ref.htm">
+    <meta name="assert" content="Test passes if this figure and the decoration are idencial">
```

Comment:
This should say someting like emphasis marks apply to the composed character but the each character inside tate-chu-yoko.